instances/widgets.treasury-factory.near/widget/pages/payments/CreatePaymentRequest.jsx

```diff
@@ -305,7 +305,7 @@ function onSubmitClick() {
   setTxnCreated(true);
   const isNEAR = tokenId === tokenMapping.NEAR;
   const gas = 270000000000000;
-  const deposit = daoPolicy?.proposal_bond || 100000000000000000000000;
+  const deposit = daoPolicy?.proposal_bond || 0;
```

Comment:
should proposal_bond be a string?
<img width="1448" alt="image" src="https://github.com/user-attachments/assets/9d75e9f5-50d9-4da1-aeec-9e75fb11b492" />